src/main/webapp/scripts/poller.js

```diff
@@ -47,22 +41,30 @@
    * @private
    */ 
   poll_() {
-    throw new Error('Unimplemented');
+    this.result_ = this.pollingFunction_();
```

Comment:
Please create all instance variables in the constructor, with initial values, before you use them elsewhere